src/vpn-worker.ts

```diff
@@ -105,63 +105,62 @@ const worker = async (instanceId: number, serviceId: number) => {
 	});
 
 	const drainConnections = async () => {
-		const clientCount = Object.keys(clientCache).length;
+		let clientCount = Object.keys(clientCache).length;
```

Comment:
I think it makes more sense to grab the values here
```suggestion
		const clients = Object.values(clientCache);
```
and then use either `clients.length` or `const clientCount = clients.length` and also reuse it as
```
for (const cn of clients) {
```
below to avoid repeating the work